blueocean-core-js/package.json

```diff
@@ -43,6 +43,7 @@
     "jsonwebtoken": "7.1.9",
     "mobx": "2.6.0",
     "mobx-utils": "1.1.2",
+    "moment": "2.15.1",
```

Comment:
bin/checkdeps.js should take care of this issue, it's been updated with the headers to also check core-js because I found getting the wrong JDL in there lets the smoke out.